Automattic/themes
# Blank Canvas Blocks: Add a build tool

#### Changes proposed in this Pull Request:
This adds a simple build tool which combines the experimental-theme.json file from Blank Canvas Blocks with the `child-theme.json` file in a child theme. This will make it easier to build child themes when the parent theme needs to change.

To test, install this child theme: 
[child.zip](https://github.com/Automattic/themes/files/6212357/child.zip)

Delete the experimental-theme.json file from the child theme.
Then run `npm run build-child child` from the blank-canvas-blocks directory.
Check that the child theme has an experimental-theme.json file.
